packages/web-components/fast-foundation/src/utilities/idle-callback-queue.ts

```diff
@@ -0,0 +1,97 @@
+/**
+ *  A service to batch idle callbacks
+ *
+ * @public
+ */
+export class IdleCallbackQueue {
+    /**
+     * Timeout value used in idle callback requests (milliseconds)
+     * (ie. max delay before a callback regardless of system load)
+     *
+     * @internal
+     */
+    public idleCallbackTimeout: number = 1000;
+
+    /**
+     * Interval in ms between getting an idle callback and requesting another.
+     * (once per 60fps frame)
+     *
+     * @internal
+     */
+    private idleCallbackInterval: number = 20;
+
+    private callBackQueue: Map<Element, () => void> = new Map<Element, () => void>();
```

Comment:
Seems to be the only variable with this casing - capital on the "B".

Reply:
done